/RoboCollectorGuiRos2ParamProvider:
  ros__parameters:
    use_sim_time: false
    gui_window_x: 72
    gui_window_y: 27
    gui_window_width: 1848
    gui_window_height: 1053
    
    #supported executor types:
       #SINGLE_THREADED(0)
       #STATIC_SINGLE_THREADED(1)
       #MULTI_THREADED(2)
    ros2_executor_type: 2
    
    #default value of 0 will use number of cpu threads
    #used only for MULTI_THREADED executor
    ros2_executor_threads_num: 4
    
    total_game_seconds: 180
    target_win_coins: 30
    use_local_controller_mode: True